.github/workflows/build.yml

```diff
@@ -0,0 +1,65 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+  schedule:
+    - cron:  '0 0 * * *'
+
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        scalaVersion: ['2.12.10', '2.13.1']
+        javaTag: ['8u222', '11.0.4', '11.0.2-oraclelinux7', 'graalvm-ce-19.3.0-java11', 'graalvm-ce-19.3.0-java8']
+        include:
+          - javaTag: '8u222'
+            dockerContext: 'debian'
+            baseImageTag: '8u222-jdk-stretch'
+          - javaTag: '11.0.4'
+            dockerContext: 'debian'
+            baseImageTag: '11.0.4-jdk-stretch'
+          - javaTag: '11.0.2-oraclelinux7'
+            dockerContext: 'oracle'
+            baseImageTag: '11.0.2-jdk-oraclelinux7'
+          - javaTag: 'graalvm-ce-19.3.0-java11'
+            dockerContext: 'graalvm-ce'
+            baseImageTag: '19.3.0-java11'
+          - javaTag: 'graalvm-ce-19.3.0-java8'
+            dockerContext: 'graalvm-ce'
+            baseImageTag: '19.3.0-java8'
+    steps:
+    - uses: actions/checkout@v1
+    - name: Get latest SBT version
+      id: get_sbt_version
+      run: |
```

Comment:
What would the process be when a new sbt version is released? I guess we could as well put it as a variable?

Reply:
The process kind of looks like this:
1. released a new sbt version eg. `1.3.6`
2. the sbt team put it into a linux package repository https://dl.bintray.com/sbt/
3. in the `build.yml:L11` (`- cron:  '0 0 * * *'`) says: "run this workflow every day at midnight"
4. in the workflow the `get_sbt_version` step find out which is the latest sbt version (get all sbt debian packages metadata from here: https://dl.bintray.com/sbt/debian/Packages and filter out the latest version)

So when a new sbt version released we just have to wait one day and automatically build the new image.
But ofc we could use a "static" variable, but then we should update it after every sbt release.

Reply:
makes sense, missed that part